### DISCUSSION TITLE
Add possibility to search references for overloaded functions / classes

### DISCUSSION THREAD

**Discussion by sadovsf** [microsoft/vscode-cpptools#7088] [Ideas]
Resharper has one great feature which is that you can navigate inheritance tree of classes as well as search for overloaded implementations for specific functions similar way you can search for references. 

This is extremely useful for navigating code as you are usually get into base class / base class empty interface function and you need to search for inherited functions in a hard way manually.

**Comment by bobbrow** [comment]
I think this feature request might be tracking what you're asking for: #3799

**Reply by sadovsf** [reply]
Thats not really the same. I don't need to visualize it. In big project that would be just huge mess of classes. I need to navigate through then when programing effectively.
